Yosemite/Yosemite/Stores/PaymentGatewayAccountStore.swift

```diff
@@ -64,13 +70,34 @@ private extension PaymentGatewayAccountStore {
             case .success(let wcpayAccount):
                 let account = wcpayAccount.toPaymentGatewayAccount(siteID: siteID)
                 self.upsertStoredAccountInBackground(readonlyAccount: account)
-                onCompletion(.success(()))
-                return
-            case .failure(let error):
+            case .failure(_):
+                DDLogDebug("Error fetching WCPay account - it is possible the extension is not installed or inactive")
                 self.deleteStaleAccount(siteID: siteID, gatewayID: WCPayAccount.gatewayID)
-                onCompletion(.failure(error))
+            }
+            group.leave()
+        }
+
+        /// Stripe
+        ///
+        group.enter()
+        stripeRemote.loadAccount(for: siteID) { [weak self] result in
+            guard let self = self else {
                 return
             }
+
+            switch result {
+            case .success(let stripeAccount):
+                let account = stripeAccount.toPaymentGatewayAccount(siteID: siteID)
+                self.upsertStoredAccountInBackground(readonlyAccount: account)
+            case .failure(_):
+                DDLogDebug("Error fetching Stripe account - it is possible the extension is not installed or inactive")
+                self.deleteStaleAccount(siteID: siteID, gatewayID: StripeAccount.gatewayID)
```

Comment:
nit: some test cases on this logic would be nice 😄 for easier unit testing, there are some examples of `*RemoteProtocol` in the app where we mock the remote instances so that we don't have to generate json files for different scenarios

Reply:
Opened https://github.com/woocommerce/woocommerce-ios/issues/5867